### PR TITLE
Replace @JonasFranzDEV with @jonasfranz

### PR DESCRIPTION
**Pull request by jonasfranz**

### CODE REVIEW

File: MAINTAINERS [hunk 1]
```diff
@@ -20,7 +20,10 @@ Jonas Östanbäck <jonas.ostanback@gmail.com> (@cez81)
 David Schneiderbauer <dschneiderbauer@gmail.com> (@daviian)
 Peter Žeby <morlinest@gmail.com> (@morlinest)
 Matti Ranta <matti@mdranta.net> (@techknowlogick)
-Michael Lustfield <mtecknology@debian.org> (@MTecknology)
```
Comment (sapk) [inline MAINTAINERS:23]: Some maintainers are removed in this PR

**Review by sapk — CHANGES_REQUESTED** [review]

File: MAINTAINERS (hunk 1 above)
Comment (sapk) [reply MAINTAINERS:23]: Related go-gitea/blog#83 (comment), go-gitea/gitea#5599 and https://github.com/go-gitea/gitea/pull/4979

**Review by sapk — APPROVED** [review]

**Review by techknowlogick — APPROVED** [review]

**Review by sapk — APPROVED** [review]

**Comment by MTecknology** [comment]
This pull should *NOT* be merged. As previously stated, this repository is *NOT* for general source changes. Changes in this repository should be left up to someone that is familiar with Debian packaging.

**Comment by jonasfranz** [comment]
@MTecknology sorry, this PR was created automatically. I will close this PR.

**Pull request closed without merging**